### PR TITLE
Christina/fix pep8

### PR DESCRIPTION
**Pull request by cahrens**
@efischer19 and @dianakhuang please review when you have time-- no rush. I just couldn't resist reducing the number of violations somewhat....

Also, I found a bug with the pep8 script (see note below).

### CODE REVIEW

File: scripts/run-pep8.sh [hunk 1]
```diff
@@ -10,7 +10,7 @@ pep8 --config=.pep8 openassessment > $PEP8_VIOLATIONS
 NUM_PEP8_VIOLATIONS=$(cat $PEP8_VIOLATIONS | wc -l)
 
 echo "Found" $NUM_PEP8_VIOLATIONS "pep8 violations, threshold is" $MAX_PEP8_VIOLATIONS
-if [[ $NUM_PEP8_VIOLATIONS > $MAX_PEP8_VIOLATIONS ]]; then
+if [[ $NUM_PEP8_VIOLATIONS -gt $MAX_PEP8_VIOLATIONS ]]; then
```
Comment (cahrens) [inline scripts/run-pep8.sh]: Note to reviewers: apparently ">" does something different than greater than. It appeared to work with the number I tried, but then I hit a number that didn't work. 

I don't have experience writing bash scripts...

http://www.tldp.org/LDP/abs/html/comparison-ops.html

**Comment by dianakhuang** [comment]
:+1:

File: openassessment/xblock/submission_mixin.py [hunk 2]
```diff
@@ -435,8 +435,10 @@ def submission_path_and_context(self):
             student_submission = self.get_user_submission(
                 workflow["submission_uuid"]
             )
-            context["peer_incomplete"] = "peer" in workflow["status_details"] and not workflow["status_details"]["peer"]["complete"]
-            context["self_incomplete"] = "self" in workflow["status_details"] and not workflow["status_details"]["self"]["complete"]
+            peer_in_workflow = "peer" in workflow["status_details"]
+            self_in_workflow = "self" in workflow["status_details"]
+            context["peer_incomplete"] = peer_in_workflow and not workflow["status_details"]["peer"]["complete"]
+            context["self_incomplete"] = self_in_workflow and not workflow["status_details"]["self"]["complete"]
```
Comment (efischer19) [inline openassessment/xblock/submission_mixin.py]: Would these be "cleaner" with `get()` and default values?

`context["peer_incomplete"] = workflow["status_details"].get("peer"), {}).get("complete", False)`

I'm okay with this as it is here, just throwing a suggestion out there.

**Comment by efischer19** [comment]
:+1: One minor suggestion, feel free to ignore if desired.

Hooray for code cleanup!

**Pull request merged**